js/src/path.js

```diff
@@ -366,3 +374,7 @@ export class HashIndexPath {
     return `[#${this.hash.toString()}]${ann}`;
   }
 }
+
+function undefinedToNull(v: ?Value): ?Value {
```

Comment:
Looks like Flow got a way to distinguish these now:

https://flowtype.org/try/#0PQKgBAAgZgNg9gdzCYAoVBjOA7AzgFzAFcAuY7AEwFMoBLbKisAXjGyJhgG5McC2y7Ti3LU6DCjyA

```js
function undefinedToNull(v: ?Value): Value | null {
```